build.gradle

```diff
@@ -0,0 +1,156 @@
+buildscript {
+  ext {
+    springBootVersion = '1.5.8.RELEASE'
+  }
+  repositories {
+    mavenCentral()
+    jcenter()
+  }
+  dependencies {
+    classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+    classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3'
+  }
+}
+
+apply plugin: 'checkstyle'
+apply plugin: 'java-library'
+apply plugin: 'org.springframework.boot'
+apply plugin: 'maven'
+apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.bintray'
+
+group 'uk.gov.hmcts.reform.cmc'
+version '1.0.0'
+
+checkstyle.toolVersion = '7.2'
+checkstyle.configFile = new File(rootDir, "checkstyle.xml")
+
+tasks.withType(Checkstyle).each { checkstyleTask ->
```

Comment:
this is using the old checkstyle hack for failing on warnings check the claim store for how we do it now

Reply:
removed